test/conformance/service_test.go

```diff
@@ -33,7 +33,7 @@ import (
 )
 
 const (
-	userPort = int32(8888)
+	userPort = int32(8081)
```

Comment:
it's weird why 8888 wouldn't work, I suspect it's a flake. Was it a consistent repro?

Reply:
Not 100% consistent but 90% I'd say. All the reruns above have been because of that issue. I was clueless as well about that.